src/components/input-company.js

```diff
@@ -1,5 +1,5 @@
 /** @jsx jsx */
-import React from 'react';
+import React from "react";
```

Comment:
Many caution messages about not using `React`

Reply:
What he said ^^^ x2